### ISSUE TITLE
setting playArea greater than 1 removes all ground objects

### ISSUE THREAD

**Issue by rmrbytes** [supermedium/aframe-environment-component#37]
I have the following environment settings:

`environment="lightPosition:-2 3 -2; seed:10; groundYScale:10; groundTexture:squares; ground:hills; playArea:10; shadow:true; shadowSize:5"`

I notice the hills appear only if I set the playArea to 1. And if I do, it creates some small hills in the centre of the scene which I want to avoid. Hence set the playArea.

Thanks.

**Comment by feiss** [comment]
I guess `playArea` doesn't work as you wish. Instead of widening the center flat rectangle on the floor, it flattens the ground in a hinge fashion:
![image](https://user-images.githubusercontent.com/359872/37553870-4b8c69d0-29d0-11e8-8b42-f1ba82f402e2.png)
I'd should revisit this parameter and improve it, I'm afraid..

Thanks you!

**Comment by rmrbytes** [comment]
From your above diagram, is then there any other parameter to increase the gap where the hills start. It appears there is some gap you leave. Is that gap not parameterised? Thanks

**Comment by feiss** [comment]
It should be, but there is no parameter yet I'm afraid :)

**Comment by rmrbytes** [comment]
Hmm. ok. Kindly consider adding it to the wish list on a future update. I (and many of us) find this component very useful. Thanks

**Comment by feiss** [comment]
Yup, let's leave this issue opened until it's done

**Comment by feiss** [comment]
Ok, I think I fixed your problem. `playArea` works like before, but now ground that is very close to y=0 is forced to be 0 so we have now a complete flat play area for all values of `playArea` > 1

Does it work for you?

**Comment by rmrbytes** [comment]
I did not quite get it? What do you mean by "ground that is very close to y=0"? If I have, say, the floor of a building set at y=0.01, are you suggesting that the uneven ground will not be created, if I set the playArea say to 1?

**Comment by feiss** [comment]
You complained about some
>  (...)small hills in the centre of the scene which I want to avoid

Now you won't have those small hills, the environment-component ground is completely flat inside playArea

**Comment by rmrbytes** [comment]
Yes, yes. And the playArea remains a circle with radius R, right? Sounds good. Would like to try it. Has the change been implemented? And hills appear beyond that.

**Comment by feiss** [comment]
yes it is implemented already, you can try.

**Comment by rmrbytes** [comment]
I see no difference. I have created a glitch

https://historical-printer.glitch.me/

When I set `<a-entity environment="lightPosition:-2 3 -2; seed:10; groundYScale:10; groundTexture:squares; ground:hills; playArea:2; shadow:true; shadowSize:5"></a-entity>` there are no hills visible.

But when I change to: `<a-entity environment="lightPosition:-2 3 -2; seed:10; groundYScale:10; groundTexture:squares; ground:hills; playArea:0; shadow:true; shadowSize:5"></a-entity>` it shows.

Even works when playArea is set to 1. But 2 onwards it fails.
